turbinia/workers/__init__.py

```diff
@@ -250,6 +296,44 @@ def __init__(
     self.user = user if user else getpass.getuser()
     self._evidence_config = {}
 
+  def serialize(self):
+    """Converts the TurbiniaTask object into a serializable dict.
+
+    Returns:
+      Dict: Dictionary representing this object, ready to be serialized.
+    """
+    task_copy = deepcopy(self.__dict__)
+    task_copy['output_manager'] = self.output_manager.__dict__
+    task_copy['last_update'] = str(self.last_update)
+    return task_copy
+
+  @classmethod
+  def deserialize(cls, input_dict):
+    # TODO(eriwcz) import Task objects (like client.py)
```

Comment:
Doesn't this need to be done before this is submitted?  Could we add a quick test to make sure these methods are exercised?  Thanks!

Reply:
👍